AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.    def unskript_default_printer(output):
</pre><p>do we need this function defined here? it should be a generic definition ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
i am not sure.  i hesitate to remove it since I know it works today.